### PR TITLE
Add prompt to publish on discord

### PR DESCRIPTION
**Pull request by itsjustkevin**
Update conductor post release instructions to prompt the release engineer to publish on discord.

### CODE REVIEW

File: dev/conductor/core/lib/src/state.dart [hunk 1]
```diff
@@ -28,7 +28,7 @@ const String stablePostReleaseMsg = """
   '\t\t Best practices: ${globals.hotfixDocumentationBestPractices}',
   '\t 2. Post announcement to flutter-announce group',
   '\t\t Flutter Announce: ${globals.flutterAnnounceGroup}',
-  '\t 3. Post announcement to discord',
+  '\t 3. Post announcement to discord and press the publish button.',
```
Comment (Hixie) [inline dev/conductor/core/lib/src/state.dart:31]: trivial nit: you have a period on this one but not the other one (and it looks like not on the other lines?)

**Review by Hixie — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
test-exempt: configuration change

File: dev/conductor/core/lib/src/state.dart (hunk 1 above)
Comment (itsjustkevin) [reply dev/conductor/core/lib/src/state.dart]: Fixed!

**Review by XilaiZhang — APPROVED** [review]

**Pull request merged**